src/pages/clients-corporate-list/forms/ClientTemplateForm.js

```diff
@@ -335,47 +319,38 @@ const ClientTemplateForm = ({
       setErrorMessage={setErrorMessage}
     >
       <Grid container>
-        <Grid container xs={12} spacing={2} sx={{ padding: "30px" }}>
-          <Grid item xs={6} sx={{ padding: "40px" }}>
+        <Grid container xs={12} spacing={2} sx={{ padding: '30px' }}>
+          <Grid item xs={6} sx={{ padding: '40px' }}>
             <Grid container spacing={3}>
               <Grid item xs={12}>
                 <TextFieldReference
-                  name="reference"
+                  name='reference'
                   label={_labels.reference}
                   value={formik.values?.reference}
                   endpointId={CurrencyTradingSettingsRepository.Defaults.get}
-                  param={"ct-nra-corporate"}
+                  param={'ct-nra-corporate'}
                   setReferenceRequired={setReferenceRequired}
                   onChange={formik.handleChange}
-                  maxLength="10"
+                  maxLength='10'
                   editMode={editMode}
-                  onClear={() => formik.setFieldValue("reference", "")}
-                  error={
-                    formik.touched.reference && Boolean(formik.errors.reference)
-                  }
-                  helperText={
-                    formik.touched.reference && formik.errors.reference
-                  }
+                  onClear={() => formik.setFieldValue('reference', '')}
+                  error={formik.touched.reference && Boolean(formik.errors.reference)}
+                  helperText={formik.touched.reference && formik.errors.reference}
```

Comment:
please remove all helperText from the fields, when a field is mandatory it is enough to have a red border, no need for the message